owncloudApp/src/main/res/layout/account_action.xml

```diff
@@ -1,8 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
   ownCloud Android client application
 
-  Copyright (C) 2016 ownCloud GmbH.
+  @author Javier Rodríguez Pérez
```

Comment:
In XML files we don't usually add the author like in Kotlin files 👍

Reply:
Sorry about that, didn't know 😇